From 92bcc4e33959e83d32d0947c6fff0654307fae82 Mon Sep 17 00:00:00 2001
From: Liu Xiaowen <b37945@freescale.com>
Date: Thu, 29 Aug 2013 13:15:10 +0800
Subject: [PATCH 244/602] ENGR00215174 remove ro.bq.gpu_to_cpu_unsupported
 proporty.

remove ro.bq.gpu_to_cpu_unsupported proporty.
The new method to do capture screen is ok now.

Signed-off-by: Liu Xiaowen <b37945@freescale.com>
---
 arm2_6dq/init.rc            | 1 -
 arm2_6sl/init.rc            | 1 -
 evk_6sl/init.rc             | 1 -
 hdmidongle_6dq/init.rc      | 1 -
 imx53_smd/init.freescale.rc | 1 -
 sabreauto_6q/init.rc        | 1 -
 sabresd_6dq/init.rc         | 1 -
 7 files changed, 7 deletions(-)

diff --git a/arm2_6dq/init.rc b/arm2_6dq/init.rc
index dc5a161..e267994 100755
--- a/arm2_6dq/init.rc
+++ b/arm2_6dq/init.rc
@@ -17,7 +17,6 @@ on boot
     setprop debug.sf.showfps    0
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	1
-    setprop ro.bq.gpu_to_cpu_unsupported 1
 
     # fsl omx graphic manager media framework property
     setprop media.omxgm.enable-player 1
diff --git a/arm2_6sl/init.rc b/arm2_6sl/init.rc
index ff5a216..646105f 100755
--- a/arm2_6sl/init.rc
+++ b/arm2_6sl/init.rc
@@ -14,7 +14,6 @@ on boot
     setprop debug.sf.showfps    0
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	0
-    setprop ro.bq.gpu_to_cpu_unsupported 1
 
     # fsl omx graphic manager media framework property
 
diff --git a/evk_6sl/init.rc b/evk_6sl/init.rc
index 80d0ce6..934828a 100644
--- a/evk_6sl/init.rc
+++ b/evk_6sl/init.rc
@@ -31,7 +31,6 @@ on boot
     setprop debug.sf.showfps    0
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	0
-    setprop ro.bq.gpu_to_cpu_unsupported 1
 
     setprop hwc.stretch.filter  1
     setprop hwc.enable_dither   1
diff --git a/hdmidongle_6dq/init.rc b/hdmidongle_6dq/init.rc
index a70b3ad..a52334a 100755
--- a/hdmidongle_6dq/init.rc
+++ b/hdmidongle_6dq/init.rc
@@ -32,7 +32,6 @@ on boot
     setprop debug.sf.showfps    0
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	1
-    setprop ro.bq.gpu_to_cpu_unsupported 1
 
     setprop hwc.stretch.filter  1
     setprop hwc.enable_dither   1
diff --git a/imx53_smd/init.freescale.rc b/imx53_smd/init.freescale.rc
index f4708e7..0bc9475 100755
--- a/imx53_smd/init.freescale.rc
+++ b/imx53_smd/init.freescale.rc
@@ -40,7 +40,6 @@ on boot
     setprop debug.sf.showfps    0
     setprop debug.sf.hw         1
     setprop ro.sf.lcd_density   160
-    setprop ro.bq.gpu_to_cpu_unsupported 1
 
     # fsl omx graphic manager media framework property
     setprop media.omxgm.enable-player 1
diff --git a/sabreauto_6q/init.rc b/sabreauto_6q/init.rc
index e264e86..fdf7065 100644
--- a/sabreauto_6q/init.rc
+++ b/sabreauto_6q/init.rc
@@ -32,7 +32,6 @@ on boot
     setprop debug.sf.showfps    0
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	1
-    setprop ro.bq.gpu_to_cpu_unsupported 1
 
     setprop hwc.stretch.filter  1
     setprop hwc.enable_dither   1
diff --git a/sabresd_6dq/init.rc b/sabresd_6dq/init.rc
index 3bf3d95..de323c6 100755
--- a/sabresd_6dq/init.rc
+++ b/sabresd_6dq/init.rc
@@ -37,7 +37,6 @@ on boot
     setprop debug.sf.showfps    0
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	1
-    setprop ro.bq.gpu_to_cpu_unsupported 1
 
     setprop hwc.stretch.filter  1
     setprop hwc.enable_dither   1
-- 
1.8.0

